### DISCUSSION TITLE
How to export without NODE_SKINNED_MESH_NON_ROOT Warning

### DISCUSSION THREAD

**Discussion by garyiskidding** [google/model-viewer#3238] [Q&A]
Greetings,

We have multiple files that show the warning "NODE_SKINNED_MESH_NON_ROOT"

![image](https://user-images.githubusercontent.com/10496265/156505849-a668340e-a800-4b07-bd0c-a026dd38a68d.png)

and the gltf file is just not visible:

![image](https://user-images.githubusercontent.com/10496265/156505919-0547df05-71fb-4b65-b104-dbfc9bf669bc.png)

As per this discussion thread : 

https://github.com/google/model-viewer/discussions/2051#discussioncomment-1998744
![Screenshot 2022-03-03 at 11 35 20 AM](https://user-images.githubusercontent.com/10496265/156506209-15e265e0-627b-4969-af55-296e4cb5cf1d.png)

, this "NODE_SKINNED_MESH_NON_ROOT" may be causing issues.

What is the right way to export a file from Blender/Maya etc. so that we do not get this warning?

**Comment by garyiskidding** [comment]
Sample file which has this problem. But is otherwise cleanly exported from blender.
[bear_13.zip](https://github.com/google/model-viewer/files/8175443/bear_13.zip)

**Reply by elalish** [reply]
Can you comment on what @donmccurdy was saying above? Specifically, have you tried unnesting before exporting? Do the parent nodes have transforms?

**Reply by garyiskidding** [reply]
Thanks for the reply @elalish . 

I am not really sure what is really meant by  "un-nesting in blender". It would be great if there is any example of that for me to view and follow.
